src/CivicrmBreadcrumbBuilder.php

```diff
@@ -30,9 +31,14 @@ public function __construct(TranslationInterface $stringTranslation, CivicrmPage
   /**
    * {@inheritdoc}
    */
-  public function applies(RouteMatchInterface $route_match) {
+  // public function applies(RouteMatchInterface $route_match) {
+  public function applies(RouteMatchInterface $route_match, ?CacheableMetadata $cacheable_metadata = NULL) {
     $route_object = $route_match->getRouteObject();
 
+    // @todo Remove in Drupal 12.0.0, will be added from ::applies(). See
+    //   https://www.drupal.org/project/drupal/issues/3459277
+    $route_object->addCacheContexts(['route']);
```

Comment:
Hi @gitressa it's not the route object you want to use here, it's the $cacheable_metadata. But note it might be null, so you'll need to check if it's null first.

Reply:
Thanks for the feedback. I simply copied the bits from another patch, and not too sure how to proceed ...

Maybe you can add you tip as a suggestion, and then I'll approve it? Thanks!

Reply:
Thanks for the suggestion, which works perfectly. I no longer get any errors with the new code. And yes, I use UI on Github ...

Reply:
It looks like the merger can Squash on commit:

- https://github.blog/open-source/git/squash-your-commits/
- https://www.reddit.com/r/github/comments/1bhnwj0/how_to_squash_all_commits_of_a_pr_via_github_web/